### PR TITLE
Update HotShot to 0.5.11

### PR DESCRIPTION
**Pull request by tbro**
updates tag version and does a `cargo update`

### CODE REVIEW

**Comment by tbro** [comment]
Addresses: https://github.com/EspressoSystems/espresso-sequencer/issues/1074

**Review by nomaxg — APPROVED** [review]

File: src/data_source.rs [hunk 1]
```diff
@@ -636,7 +636,7 @@ pub mod status_tests {
         {
             let ds = ds.read().await;
             // Check that block height is initially one (for the genesis block).
-            assert_eq!(ds.block_height().await.unwrap(), 1);
+            assert_eq!(ds.block_height().await.unwrap(), 0);
             // With consensus paused, check that the success rate returns infinity (since the block
             // height, the numerator, is 1, and the view number, the denominator, is 0).
```
Comment (jbearer) [inline src/data_source.rs:641]: Comments are out of date now

File: src/status.rs [hunk 2]
```diff
@@ -172,7 +172,7 @@ mod test {
             sleep(Duration::from_secs(1)).await;
         }
         // The block height is initially 1 (for the genesis block).
```
Comment (jbearer) [inline src/status.rs:174]: Comment is out of date now

**Review by jbearer — CHANGES_REQUESTED** [review]

File: src/data_source.rs (hunk 1 above)
Comment (tbro) [reply src/data_source.rs:641]: on it

File: src/status.rs (hunk 2 above)
Comment (tbro) [reply src/status.rs]: fixed in 14883ca

File: src/data_source.rs (hunk 1 above)
Comment (tbro) [reply src/data_source.rs]: fixed in 14883ca

File: src/data_source.rs [hunk 3]
```diff
@@ -635,11 +635,11 @@ pub mod status_tests {
 
         {
             let ds = ds.read().await;
-            // Check that block height is initially one (for the genesis block).
+            // Check that block height is initially zero (for the genesis block).
```
Comment (jbearer) [inline src/data_source.rs:638]:
```suggestion
            // Check that block height is initially zero.
```

File: src/status.rs [hunk 4]
```diff
@@ -171,7 +171,7 @@ mod test {
             );
             sleep(Duration::from_secs(1)).await;
         }
-        // The block height is initially 1 (for the genesis block).
+        // The block height is initially zero (for the genesis block).
```
Comment (jbearer) [inline src/status.rs:174]:
```suggestion
        // The block height is initially zero.
```

**Review by jbearer — APPROVED** [review]
Looks good, pending CI 🤞

**Pull request merged**